### PR TITLE
Searchable Glossary 

### PR DESCRIPTION
**Pull request by ServiceDog**
Task details: https://app.dework.xyz/arrow-air/general-tasks?taskId=943e5d21-ae5b-4426-a1ef-aace9b8619f4

There are multiple options for implementing this. Please see the DeWork above for explanation.

This push does the quick and dirty first option, which may not scale well with thousands of entries

### CODE REVIEW

**Comment by owlot** [comment]
I think for the "quick and dirty" way, we should be able to implement one of the existing local search plugins of Docusaurus:
https://docusaurus.io/community/resources#search
They support offline/local search, so don't add any dependency on a database and integrate a search bar for the entire website.

Once our website starts to grow, we probably do want to switch to same search backend for this.

**Comment by ServiceDog** [comment]
> I think for the "quick and dirty" way, we should be able to implement one of the existing local search plugins of Docusaurus: https://docusaurus.io/community/resources#search They support offline/local search, so don't add any dependency on a database and integrate a search bar for the entire website.
> 
> Once our website starts to grow, we probably do want to switch to same search backend for this.

Noted, replaced with docusaurus local search

searchbar

**Review by owlot — DISMISSED** [review]

File: docs/documentation/glossary.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    /* font-family: Karla sans-serif; */
+    min-width: 400px;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0);
+    border-radius: 10px;
+}
+
+.glossaryTerm {
+    font-size: 20%;
+    font-weight: bold;
+}
```
Comment (GoodluckH) [inline docs/documentation/glossary.css:42]: Are these being used at all? If not, please remove.

**Review by GoodluckH — CHANGES_REQUESTED** [review]
Generally LGTM, we might need to rid of unused codes.

File: docs/documentation/glossary.css (hunk 1 above)
Comment (ServiceDog) [reply docs/documentation/glossary.css]: Fixed in https://github.com/Arrow-air/website/pull/44/commits/23f391a6292061ac0c46f1a29b6e161e59b0ee66

**Review by GoodluckH — APPROVED** [review]

**Pull request merged**